Core/include/Acts/Seeding2/DoubletSeedFinder.hpp

```diff
@@ -180,6 +180,10 @@ struct MiddleSpInfo {
   float cosPhiM{};
   /// ratio between middle SP y position and radius
   float sinPhiM{};
+  /// deltaZ maximum for bottom doublet
+  float maxDeltaZbottom{};
+  /// deltaZ maximum for top doublet
+  float maxDeltaZtop{};
```

Comment:
sorry I think I miscommunicated this - I believe this should be separate from the `MiddleSpInfo`